package.json

```diff
@@ -12,24 +12,28 @@
   "bugs": {
     "url": "https://github.com/WalletConnect/walletconnect-utils/issues"
   },
+  "workspaces": [
+    "browser/*",
+    "jsonrpc/*",
+    "crypto/*",
+    "misc/*",
+    "relay/*"
+  ],
   "scripts": {
-    "clean": "lerna clean --yes",
-    "bootstrap": "lerna bootstrap",
-    "lint": "lerna run lint",
-    "prettier": "lerna run prettier",
-    "format": "lerna run format",
-    "build": "lerna run build",
-    "test": "lerna run test",
-    "check": "run-s prettier lint build test",
+    "lint": "turbo lint",
+    "prettier": "turbo prettier",
+    "format": "turbo format",
+    "build": "turbo build",
+    "test": "turbo test",
+    "check": "turbo run prettier lint build test",
+    "check:force": "turbo run prettier lint build test --force=true",
     "reset": "run-s clean bootstrap check",
```

Comment:
```suggestion
```
I think this one is a leftover command from lerna as well

Reply:
Yup good catch thank you!